src/Tizen.Multimedia.Remoting/MediaController/MediaControlCommand.cs

```diff
@@ -59,10 +60,22 @@ internal void SetRequestInformation(string receiverId)
         /// <param name="requestId">The request Id for each command.</param>
         internal void SetResponseInformation(string receiverId, string requestId)
         {
-            ReceiverId = receiverId ?? throw new ArgumentNullException(nameof(receiverId)); ;
-            _requestId = requestId ?? throw new ArgumentNullException(nameof(requestId)); ;
+            ReceiverId = receiverId ?? throw new ArgumentNullException(nameof(receiverId));
+            _requestId = requestId;
+
+            if (_requestId == null)
+            {
+                Log.Info(GetType().FullName, "request_id is null. Response() should not be called.");
+            }
         }
 
+        /// <summary>
+        /// Gets the status of response.
```

Comment:
how it sounds suggests that it can change, like when Response is supplied it would be set to false then,
suggestion: "Indicates whether a response is expected for this command." or "Indicates if the command should be responded to" or shorter brief "Indicates if the response is needed."

Reply:
I'll change it to 'Indicates if the response is needed.'